feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go

```diff
@@ -117,3 +117,52 @@ func findProcessByName(ctx context.Context, t *testing.T, dut *ondatra.DUTDevice
 	}
 	return pID
 }
+
+func TestChassisComponentArtifacts(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+	gnoiClient := dut.RawAPIs().GNOI().New(t)
+	componentName := map[string]string{"name": components[dut.Vendor()]}
+	// Execute Healthz Check RPC for the chassis component.
+	chkReq := &hpb.CheckRequest{
+		Path: &tpb.Path{
+			Elem: []*tpb.PathElem{
+				{
+					Name: "components",
+				},
+				{
+					Name: "component",
+					Key:  componentName,
+				},
+			},
+		},
+	}
+	t.Logf("Executing Healthz Check RPC for component %v", componentName["name"])
+	validResponse2, err2 := gnoiClient.Healthz().Check(context.Background(), chkReq)
```

Comment:
Nit: it is idiomatic in golang to reuse the same `err` variable, i.e. no need to create err2, ..., errX

Also you may want to shorten/use more meaningful names for response vars, e.g. `chkRes` pairs nicely with `chkReq`, `artRes` with `artReq`, etc